File: unit3/function_demo.c
#include <cs50.h>
#include <stdio.h>

int add_two_numbers(int, int);

int main(void)
{
    int num1 = get_int("Num 1: ");
    int num2 = get_int("Num 2: ");
    int sum = add_two_numbers(num1, num2);
    printf("%d + %d = %d\n", num1, num2, sum);
}

int add_two_numbers(int x, int y)
{
    return x + y;
}

File: unit3/function_demo.py
import cs50

def add_two_numbers(x, y):
    return x + y

def main():
    num1 = cs50.get_int("Num 1: ")
    num2 = cs50.get_int("Num 2: ")
    sum = add_two_numbers(num1, num2)
    print(f"{num1} + {num2} = {sum}")

if __name__ == "__main__":
    main()
